src/components/blog/blogs/blog-item.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
```

Comment:
@UzeeChamp1 component name should start with Capital letter and shouldn't contain alphanumeric letters
```suggestion
src/components/blog/BlogItem.tsx
```

Reply:
actually the component function name is already in camel case, this is just a file name, Should I change it too?